code-of-conduct.html

```diff
@@ -47,28 +47,27 @@
     <li>Deliberate “outing” of any aspect of a person’s identity without their consent except as necessary to protect vulnerable people from intentional abuse.</li>
     <li>Publication of non-harassing private communication. </li>
   </ul>
-</p>
 
-<p>Web Matters prioritizes marginalized people’s safety over privileged people’s comfort. We reserve the right not to act on complaints regarding:
+<p>Web Matters prioritizes marginalized people’s safety over privileged people’s comfort. We reserve the right not to act on complaints regarding:</p>
 
  <ul>
     <li>‘Reverse’ -isms, including ‘reverse racism,’ ‘reverse sexism,’ and ‘cisphobia’</li>
     <li>Reasonable communication of boundaries, such as “leave me alone,” “go away,” or “I’m not discussing this with you.”</li>
     <li>Communicating in a ‘tone’ you don’t find congenial</li>
     <li>Criticizing racist, sexist, cissexist, or otherwise oppressive behavior or assumptions </li>
   </ul>
-</p>
 
-<h2>Reporting</h2>
+<h3>Reporting</h3>
 
-<p>If you are being harassed by a member of Web Matters, notice that someone else is being harassed, or have any other concerns, please contact us. If the person who is harassing you is on the team, they will recuse themselves from handling your incident. We will respond as promptly as we can.</p>
+<p>If you are being harassed by a member of Web Matters, notice that someone else is being harassed, or have any other concerns, please contact us. If the person who is harassing you is on the team, they will excuse themselves from handling your incident. We will respond as promptly as we can.</p>
```

Comment:
Given that this wording comes from someone else's CoC (Geek Feminism, I think Orde said) and either word has broadly the same meaning, perhaps we could take out this change for now in order to get the rest of the fixes merged?